ex1.py

```diff
@@ -1,3 +1,13 @@
-if __name__ == '__main__':
-    print("5")
-    print("5")
\ No newline at end of file
+
+
+def avg():
+    counter = 0
+    sum = 0
+    num = input("enter num if you want to stop enter q ")
```

Comment:
also here use const.
read about string formating( example: f"{val}")

Reply:
I did not understand how formating( example: f"{val}") is related

Reply:
in you code there are two places you use the q letter.
now if you would like to change the letter from q to b you would have to change it in 2 different places - thats a bad habbit.
instead use here the const you allready added to your code. to use it as part of a string you will need string formating